nccs/pipeline/direct/business_interruption.py

```diff
@@ -1,14 +1,33 @@
-from pathlib import Path
-import pandas as pd
-import numpy as np
 import logging
+import os
+from pathlib import Path
 
+import numpy as np
+import pandas as pd
+import pycountry
 from climada.entity import ImpactFunc
-from nccs.utils.folder_naming import get_resources_dir
+
 from nccs.pipeline.direct.combine_impact_funcs import ImpactFuncComposable
+from nccs.utils.folder_naming import get_resources_dir
 
-SECTOR_BI_DRY_PATH = Path(get_resources_dir(), 'impact_functions', 'business_interruption', 'TC_HAZUS_BI_industry_modifiers_v2.csv')
-SECTOR_BI_WET_PATH = Path(get_resources_dir(), 'impact_functions', 'business_interruption', 'FL_HAZUS_BI_industry_modifiers.csv')
+SECTOR_BI_DRY_PATH = Path(
+    get_resources_dir(),
+    'impact_functions',
+    'business_interruption',
+    'TC_HAZUS_BI_industry_modifiers_v2.csv'
+)
+SECTOR_BI_WET_PATH = Path(
+    get_resources_dir(),
+    'impact_functions',
+    'business_interruption',
+    'FL_HAZUS_BI_industry_modifiers.csv'
+)
+SECTOR_BI_WET_SCALE_PATH = Path(
+    get_resources_dir(),
+    'impact_functions',
+    'business_interruption',
+    'bi_scaling_regional.csv'
```

Comment:
I don't have this file locally – can you add it to the repo?

Reply:
sure, sorry about that. I added the file to the git